src/pages/add-item/components/AddItemForm.jsx

```diff
@@ -0,0 +1,190 @@
+          ...prev,
+          tag: [...prev.tag, value],
+        }));
+        setTagInput('');
+      }
+    }
+  };
+
+  const handleRemoveTag = (tagToRemove) => {
+    setFormData((prev) => ({
+      ...prev,
+      tag: prev.tag.filter((tag) => tag !== tagToRemove),
+    }));
+  };
+
+  const imgOnClick = () => {
+    fileInputRef.current?.click();
+  };
+
+  return (
+    <div className='p-4'>
+      <form>
+        <div className='flex justify-between items-center mb-4'>
+          <p className='text-[2rem] font-bold leading-none'>상품 등록하기</p>
+          <button
+            type='submit'
+            disabled={!isSubmitable}
+            className={`h-[4.4rem] px-[2rem] rounded text-[1.6rem] font-[400] ${
+              isSubmitable
+                ? 'bg-[#3692ff] text-white cursor-pointer'
+                : 'bg-[#9ca3af] text-gray-600 cursor-not-allowed'
+            }`}
+          >
+            등록
+          </button>
+        </div>
+
+        <div className='mb-4 mt-6'>
+          <p className='mb-2 font-[700] text-[2rem] text-[#1F2937]'>
+            상품 이미지
+          </p>
+          <div className='flex gap-4'>
+            <div
+              onClick={imgOnClick}
+              className='w-[16.8rem] h-[16.8rem] sm:w-[28.2rem] sm:h-[28.2rem] bg-[#f3f4f6] rounded flex flex-col items-center justify-center cursor-pointer'
+            >
+              <div className='text-[#9ca3af] text-[2rem] mb-1'>+</div>
+              <div className='text-[#9ca3af] text-[1.6rem]'>이미지 등록</div>
+            </div>
+
+            {previewImg && (
+              <div className='relative w-[16.8rem] h-[16.8rem] sm:w-[28.2rem] sm:h-[28.2rem]'>
+                <img
+                  src={previewImg}
+                  alt='preview'
+                  className='w-full h-full object-cover border rounded'
+                />
+                <button
+                  type='button'
+                  onClick={() => {
+                    setPreviewImg(null);
+                    setProductImg(null);
+                    setFormData((prev) => ({ ...prev, img: null }));
+                    if (fileInputRef.current) fileInputRef.current.value = '';
+                  }}
+                  className='absolute top-2 right-2 bg-gray-200 rounded-full w-6 h-6 flex items-center justify-center'
+                >
+                  <span className='text-gray-600 text-lg'>×</span>
+                </button>
+              </div>
+            )}
+          </div>
+
+          <input
+            type='file'
+            accept='image/*'
+            ref={fileInputRef}
+            onChange={handleImgUpload}
+            style={{ display: 'none' }}
+          />
+        </div>
+
+        <div className='flex flex-col gap-10'>
+          <Input
+            type='text'
+            placeholder='상품명을 입력해주세요'
+            inputName='상품명'
+            name='name'
+            value={formData.name}
+            onChange={handleChange}
```

Comment:
💊 제안
지금처럼 상품명을 useState로 제어하면 사용자가 입력할 때마다 컴포넌트가 리렌더링됩니다.
지금 코드상에서는 상품명을 바꿔주는 것이 아니라 제어 컴포넌트로 관리할 필요가 없을 것 같아요!
이를 비제어 컴포넌트로 바꾸셔서 불필요한 리렌더링을 줄이시는 것을 추천드려요!

```suggestion
            onChange={handleChange}
```